ibm/service/iampolicy/data_source_ibm_iam_trusted_profile_policy_test.go

```diff
@@ -244,7 +244,7 @@ func testAccCheckIBMIAMTrustedProfilePolicyDataSourceTimeBasedCustom(name string
 		profile_id = ibm_iam_trusted_profile.profileID.id
 		roles  = ["Viewer"]
 		resources {
-			service = "kms"
```

Comment:
why we changed exisiting testcase can we add new testcase for "service_group_id"  for this new feature.
Don't modify exisitng testcases we will not have testcase support for old features